server/archive.api.js

```diff
@@ -0,0 +1,52 @@
+const HomebrewModel = require('./homebrew.model.js').model;
+const router = require('express').Router();
+const asyncHandler = require('express-async-handler');
+
+const archive = {
+	archiveApi : router,
+	/* Searches for matching title, also attempts to partial match */
+	findBrews  : async (req, res, next)=>{
+		try {
+			const title = req.query.title || '';
+			const page = parseInt(req.query.page) || 1;
+			console.log('trying page ', page);
+			const pageSize = 10; // Set a default page size
```

Comment:
Change to `const pageSize =  parseInt(req.query.size) || 10;`.

It can remain a "hidden" feature (as an aid in testing), with _maybe_ a UI element added later.

Probably best to do this, just in case we don't want users supplying a negative page size
```
const minPageSize = 1; 
const pageSize = Math.max(parseInt(req.query.size) || 10, minPageSize);
```

Probably best to have a more reasonable minumum page size too .. otherwise we might end up with a huge mass of pagination links (e.g. `pageSize = 1`).

Reply:
That is a good idea, but i am not sure on how to implement the pagesize control, and i don't think we need it.
I'm trying to come up with a website that does something like that.

Reply:
Done, hope you like how i did it.